index.html

```diff
@@ -2968,12 +2969,305 @@ <h2>Protocol Binding</h2>
+              The resolved value of the <code>href</code> member MUST then be
+              used as the URL of the <code>Property</code> resource.</span>
+          </p>
+          <p>
+            <span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-observeproperty-3">
+              In order to observe a property, a Consumer MUST
+              provide the listener URL in the request payload of the
+              <code>observe</code> operation of the <code>Property</code> resource.</span>
+          </p>
+          <p>
+            This involves the Consumer sending an HTTP request to
+            the Web Thing with:
+          </p>
+          <ul>
+            <li>Method set to <code>POST</code></li>
+            <li>URL set to the URL of the <code>Property</code> resource</li>
+            <li>
+              <code>Accept</code> header set to <code>application/json</code>
+            </li>
+            <li>
+              A body with a subscription request payload, serialized in JSON.
+            </li>
+            <p>
+              The subscription payload contains the URI for the event message listener
+              in the field with the <code>callbackURI</code> key.
+            </p>
+          </ul>
+          <pre class="example">
+              POST /things/lamp/properties/level HTTP/1.1
+              Host: mythingserver.com
+              Content-Type: application/json
+              Accept: application/json
+              {
+                callbackURI: "http://myConsumer.com/myEventListener"
+              }
+            </pre>
+  
+          <p><span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-observeproperty-5">
+              If a Web Thing receives an HTTP request following the format
+              above and the Consumer has permission to observe the
+              corresponding property, then it MUST send <em>event messages</em>
+              to the Consumer each time the value of the specified property changes.</span>
+          </p>
+  
+          <p>
+            This involves the Web Thing initially sending an HTTP
+            response to the Consumer with:
+          </p>
+          <ul>
+            <li>Status code set to <code>200</code></li>
+            <li>
+              <code>Content-Type</code> header set to
+              <code>application/json</code>
+            </li>
+            <li>
+              <code>Accept</code> header set to <code>application/json</code>
+            </li>
+          </ul>
+          <pre class="example">
+              HTTP/1.1 200 OK
+              Content-Type: application/json
+              {
+                  subscriptionId: 1234-4544-1211
+              }
+          </pre>
+          <p><span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-observeproperty-6">
+              Whenever an event of the specified type occurs, the Web Thing MUST
```

Comment:
what is meant by event of the specified type?

Also the assertion id seems to indicate property (mentioning observeproperty) but text says events. They are sometimes similar but the semantics behind are different.

Reply:
Thanks for noticing - That's a copy paste error - fixed